Gem/Code/Source/SampleComponentManager.cpp

```diff
@@ -143,6 +144,9 @@ namespace Platform
 
 namespace AtomSampleViewer
 {
+    AZ_CVAR(bool, r_EnableDefaultRenderPipelineOnXR, false, nullptr, AZ::ConsoleFunctorFlags::Null,
+        "When an XR system is present this will enable the regular render pipeline on host PC as well (false by default).");
```

Comment:
This is good enough for now since we are supporting XR via ASV samples at the moment. But when we have support for XR at launcher level this will have to be handled generally.